test-harness/tests/test_master.py

```diff
@@ -19,45 +19,45 @@ def test_if_exhibitor_endpoint_redirects_req_without_slash(
         assert r.status_code == 301
 
     def test_if_exhibitor_endpoint_handles_redirects_properly(
-            self, master_ar_process, mocker, valid_user_header):
+            self, master_ar_process, mocker, superuser_user_header):
         location_sent = 'http://127.0.0.1/exhibitor/v1/ui/index.html'
         location_expected = 'http://127.0.0.1/exhibitor/exhibitor/v1/ui/index.html'
         mocker.send_command(endpoint_id='http://127.0.0.1:8181',
                             func_name='always_redirect',
                             aux_data=location_sent)
 
         url = master_ar_process.make_url_from_path("/exhibitor/v1/ui/index.html")
-        r = requests.get(url, allow_redirects=False, headers=valid_user_header)
+        r = requests.get(url, allow_redirects=False, headers=superuser_user_header)
 
         assert r.status_code == 307
         assert r.headers['Location'] == location_expected
 
     def test_if_request_is_sent_to_correct_upstream(self,
                                                     master_ar_process,
-                                                    valid_user_header):
+                                                    superuser_user_header):
```

Comment:
I tried to be consistent with `valid_user_header`.

What do you think about renaming it to `root_user_header`?